Lib/logging/__init__.py

```diff
@@ -160,9 +160,9 @@ def addLevelName(level, levelName):
         _releaseLock()
 
 if hasattr(sys, '_getframe'):
-    currentframe = lambda: sys._getframe(3)
+    _currentframe = lambda: sys._getframe(1)
```

Comment:
I am not sure if renaming this now is good because probably there is code out there using this, and this would break and force them up to add the infamous `if python version < xxx`.

Reply:
I agree and restored the original name. However, I do think either renaming `currentframe` to `_currentframe` or documenting it as an official method should be considered.